engine/src/main/scala/cromwell/engine/backend/local/LocalBackendCall.scala

```diff
@@ -23,7 +21,9 @@ case class LocalBackendCall(backend: LocalBackend,
   override def execute(implicit ec: ExecutionContext) = backend.execute(this)
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
-    backend.useCachedCall(avoidedTo.asInstanceOf[LocalBackendCall], this)
+    backend.useCachedCall(avoidedTo.jobDescriptor, jobDescriptor)
```

Comment:
This should not be here, the implementation in `BackendCall` should be handling this.